binding/format/protobuf/v2/pb/cloudevent.pb.go

```diff
@@ -7,7 +7,7 @@
 package pb
 
 import (
-	any "github.com/golang/protobuf/ptypes/any"
+	any1 "github.com/golang/protobuf/ptypes/any"
```

Comment:
these were automatically made as part of generation -- I installed the matching versions of `protoc` and `protoc-gen-go` as detailed at the top of this file. I would imagine these changes were made automatically because I have a newer version of go installed locally (`go1.20.7`) and the `any` alias conflicts with the `any` keyword. I'm happy to generate with 1.17 if preferred, though it might be desirable to go ahead and update these.

Reply:
Yeah, we're still including `1.17` to make sure older versions of Go can use the SDK/we don't accidentally start using `>go1.17` features (until we decide to bump).

IMHO the `any1` is not a breaking change here, but if you don't mind please re-gen with `1.17` so we don't introduce this change?

Reply:
> IMHO the any1 is not a breaking change here, but if you don't mind please re-gen with 1.17 so we don't introduce this change?

Sounds good, will do!